x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/use_monitor_breadcrumbs.test.tsx

```diff
@@ -27,14 +27,13 @@ describe('useMonitorBreadcrumbs', () => {
       return <>Step Water Fall</>;
     };
 
-    mountWithRouter(
+    renderTLWithRouter(
       <MountWithReduxProvider>
-        <KibanaContextProvider services={{ ...core }}>
-          <Route path={OVERVIEW_ROUTE}>
-            <Component />
-          </Route>
-        </KibanaContextProvider>
-      </MountWithReduxProvider>
+        <Route path={OVERVIEW_ROUTE}>
+          <Component />
+        </Route>
+      </MountWithReduxProvider>,
+      { kibanaProps: { services: { ...core } } }
```

Comment:
This is an example of overwriting the entire Kibana context core. This is useful in this context to get access to a function for getting the breadcrumbs out of core.